.github/CODEOWNERS

```diff
@@ -602,6 +602,7 @@ pep-0721.rst  @encukou
 pep-0722.rst  @pfmoore
 pep-0723.rst  @AA-Turner
 pep-0725.rst  @pradyunsg
+pep-0755.rst  @tiangolo @JelleZijlstra
```

Comment:
It should be just me, only people with commit access can be CODEOWNERS. I'll make sure you get CC'ed on any changes to the PEP though.

Reply:
Get it! Done, thank you! ✅